builder/ebs/fast_launch_setup.go

```diff
@@ -0,0 +1,106 @@
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type FastLaunchConfig
+
+package ebs
+
+import "fmt"
+
+// FastLaunchConfig is the configuration for setting up fast-launch for Windows AMIs
+//
+// NOTE: requires the Windows image to be sysprep'd to enable fast-launch. See the
+// AWS docs for more information:
+// https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/win-ami-config-fast-launch.html
+type FastLaunchConfig struct {
+	// Configure AMI with fast-launch on Windows AMIs
```

Comment:
nit: The double use of AMI tripped me up a little. What do you think of this change? 
```suggestion
	// Configure fast-launch for Windows AMIs
```

Reply:
The rework sounds good to me, good idea removing tautologies like these.